solutions/standard/DA-types.md

```diff
@@ -0,0 +1,145 @@
+# IBM Cloud Elasticsearch Deployable Architecture Variables Documentation
+This document provides detailed information about the complex object types used in the IBM Cloud Elasticsearch DA (Deployable Architecture).
+
+
+## 1. Service Credential Names
+
+**Description:**
+This variable defines a map of service credential names and their corresponding roles for the Elasticsearch database. This allows for the creation of service credentials with specific roles, providing controlled access to the database.
+Reference provided for [Account Service Credentials](https://cloud.ibm.com/docs/account?topic=account-service_credentials&interface=ui).
+
+**Type:**
+The `service_credential_names` variable is a map where the key is the name of the service credential and the value is the role assigned to that credential.
+
+**Default Value:**
+The default value for the `service_credential_names` variable is an empty map (`{}`).
+
+- `Key (name, required):` The name of the service credential.
+
+- `Value (role, required):` The role assigned to the service credential.
+
+**Sample Examples:**
+
+```json
+  {
+    "elastic_user_1" = "Viewer"
+  }
+```
+```json
+  {
+    "elastic_user_2" = "Editor"
+  }
+```
```

Comment:
can you just give 1 map that has two roles in it?

Reply:
done